packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -194,7 +195,11 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         className={mergeStyles({ width: `calc(100% - ${menuButtonContainerStyle.width})`, alignItems: 'center' })}
       >
         {avatar}
-        {me && <Stack className={meTextStyle}>{isMeText}</Stack>}
+        {me && (
+          <Stack>
```

Comment:
Do you need this `Stack`, now that this is wrapped in a `Text`?

Reply:
Updated